src/content/contribute/index.md

```diff
@@ -73,3 +73,9 @@ You can also encourage your developers to contribute to the ecosystem by open-so
 To anyone else who is interested in helping our mission -- e.g. venture capitalists, government entities, digital agencies, etc. -- we would love for you to work with us, one of the top npm packages, to improve your product! Please don't hesitate to reach out with questions.
 
 <!-- add slides here -->
+
+## Pull requests
+
+Documentation of webpack features and changes is now being updated as webpack evolves. An automated issue creation integration was established and proven to be effective since year 2018.
```

Comment:
```suggestion
Documentation of webpack features and changes is now being updated as webpack evolves. An automated issue creation integration was established and proven to be effective since the year 2018.
```

Reply:
Yeah when i read it again i think this will be better:
```suggestion
Documentation of webpack features and changes is now being updated as webpack evolves. An automated issue creation integration was established and proven to be effective in the recent years.
```